python/ray/serve/_private/deployment_state.py

```diff
@@ -2194,6 +2194,23 @@ def update(self):
             del self._deployment_states[tag]
 
         if len(deleted_tags):
+            self._record_deployment_usage()
             record_extra_usage_tag(
```

Comment:
redundant with `self._record_deployment_usage`?

Reply:
Yes, good catch! I updated it.